tox.ini

```diff
@@ -32,6 +32,7 @@ commands =
 
 [testenv:flake8]
 skip_install = True
+max-line-length=90
```

Comment:
I am actually quite fond of pep8 and its line length limitations. Normally there is always a way to reduce the line length, and should you really have a case where readability is in danger, add a `# noqa` at the end of the line.

Reply:
Slowly figuring this out, I've never used Travis before. Thanks for the guidance.